src/modules/roc_pipeline/config.h

```diff
@@ -83,7 +83,7 @@ struct SenderConfig {
     packet::channel_mask_t input_channels;
 
     //! Number of samples for internal frames.
-    size_t internal_frame_size;
+    core::nanoseconds_t internal_frame_size;
```

Comment:
Here and below, the comments should be updated accordingly, e.g.:

```
//! Duration of the internal frames, in nanoseconds.
```

Reply:
done.